woocommerce/woocommerce
# Remove IE11 specific styles

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/47901

This PR removes some of the IE specific styles.

### NOTES

- I removed only the styles that had IE comments.
- Some of the styles with IE comments that have the `-ms-` prefix -- I did check with `caniuse` to make sure it won't affect Edge browsers.
- There are still a few CSS styles with `-ms-` prefixes which I didn't remove because those could impact Edge browsers.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure the Price Slider block styles look correct on various browsers.
2. Make sure the Checkout Totals block styles look correct on various browsers.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Remove unneeded IE styling as IE is no longer supported

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
